Simple first dummy UI

da-kami opened the pull request:
![image](https://user-images.githubusercontent.com/5557790/130936728-437d5e72-8664-4a7e-bd39-3bbc5ef50b31.png)

da-kami committed:
Simple first dummy UI

klochowicz commented on frontend/src/App.test.tsx:
```diff
@@ -0,0 +1,9 @@
+    render(<App />);
+    const linkElement = screen.getByText(/learn react/i);
+    expect(linkElement).toBeInTheDocument();
+});
```
Perhaps we could remove the stale test as it wouldn't pass anyway?

klochowicz commented on frontend/src/App.tsx:
```diff
@@ -0,0 +1,98 @@
+                                    <VStack spacing={5} shadow={"md"} padding={5} align={"stretch"}>
+                                        <HStack>
+                                            <Text align={"left"}>Current Price:</Text>
+                                            <Text>490000</Text>
```
nit: that's a very optimistic price :D

klochowicz reviewed (approved):
looks like a good skeleton for the UI!

klochowicz committed:
Improve the UI by adding tiles representing a single CFD contract

Extract CFD tile as a separate component.
Place a few tiles with dummy values in the UI

itchymax commented:
bors r+
